### PR TITLE
Mike our stat tracker spec3

### PR DESCRIPTION
**Pull request by MWoodshop**
This includes fixes to the variable names in game_team and team files as well as our own stat_tracker_rspec file. This was coded live with the entire group.

### CODE REVIEW

File: lib/game_team.rb [hunk 1]
```diff
@@ -18,17 +18,17 @@ class GameTeam
...
     @goals = data[:goals]
     @shots = data[:shots]
     @tackles = data[:tackles]
     @pim = data[:pim]
-    @power_play_opp = data[:powerPlayOpportunities]
-    @power_play_goals = data[:powerPlayGoals]
-    @face_off_win_percentage = data[:faceOffWinPercentage]
+    @power_play_opp = data[:powerplayopportunities]
+    @power_play_goals = data[:powerplaygoals]
+    @face_off_win_percentage = data[:faceoffwinpercentage]
```
Comment (awiley33) [inline lib/game_team.rb:31]: Glad we figured this out!

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -58,7 +58,7 @@ def percentage_visitor_wins
 
   def percentage_ties
     tie_games = @game_teams.find_all do |game|
-      game.game_result.include?("TIE")
+      game.game_result && game.game_result.include?("TIE")
```
Comment (awiley33) [inline lib/stat_tracker.rb:79]: Good edge case coverage here

File: spec/game_team_spec.rb [hunk 3]
```diff
@@ -67,17 +67,17 @@
...
         tackles: '40',
         pim: '10',
-        powerPlayOpportunities: '4',
-        powerPlayGoals: '1',
-        faceOffWinPercentage: '10',
+        powerplayopportunities: '4',
+        powerplaygoals: '1',
+        faceoffwinpercentage: '10',
         giveaways: '20',
         takeaways: '10'
```
Comment (awiley33) [inline spec/game_team_spec.rb:82]: The mocked data looks great, nice job Mike

**Review by awiley33 — APPROVED** [review]
The test suite looks great

File: lib/stat_tracker.rb [hunk 4]
```diff
@@ -274,8 +274,38 @@ def least_accurate_team(season)
 
 
 
+    team = @teams.find do |team|
```
Comment (z-fitch) [inline lib/stat_tracker.rb:277]: What method does this go to

**Review by z-fitch — APPROVED** [review]

**Pull request merged**